src/main/java/com/megazone/goop/racingcar/StringSumCalculator.java

```diff
@@ -0,0 +1,35 @@
+package com.megazone.goop.racingcar;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringSumCalculator {
+
+    private static final String DEFAULT_DELIMITERS = "[,|:]";
+
+    private String[] split(String text) {
+        if (text == null) {
+            return new String[] {};
+        }
+
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(text);
+        if (m.find()) {
+            return m.group(2).split(DEFAULT_DELIMITERS.replace("]", "|" + m.group(1) + "]"));
+        }
+
+        return text.split(DEFAULT_DELIMITERS);
+    }
```

Comment:
코드가 짧긴하지만, 언뜻 이해하기가 쉽지 않아 보입니다.
여러 줄로 작성해서라도 하나하나의 의미를 명확히 전달하면 어떨까 합니다.

변수명과 메서드명에 의미를 부여하는 것도 가능합니다.
아래 Suggestion 코드에서 addCustomDelimiter() 메서드를 추출해서
기존 구분자에 커스텀 구분자를 추가한다는 의미를 명확히 전달해 봤습니다.

```suggestion
        String delimiters = DEFAULT_DELIMITERS;
        String realText = text;

        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(text);
        if (m.find()) {
            delimiters = addCustomDelimiter(m.group(1));
            realText = m.group(2);
        }
        return realText.split(delimiters);
    }

    private static String addCustomDelimiter(String customDelimiter) {
        return DEFAULT_DELIMITERS.replace("]", "|" + customDelimiter + "]");
    }
```

Reply:
가이드 주신대로 간결하게 코드를 수정하였습니다.